lib/streamlit/hello/hello.py

```diff
@@ -17,20 +17,24 @@
 
 from __future__ import division, unicode_literals
 
-import streamlit as st
-from streamlit.logger import get_logger
 import inspect
 from collections import OrderedDict
 import urllib
 import textwrap
 
+import streamlit as st
+from streamlit.logger import get_logger
+
+from . import util
```

Comment:
Python2.7 ok?

Reply:
This won't work with py2. But I think you can import `absolute_import` from `__future__` to make it work.